superset/models.py

```diff
@@ -1410,7 +1421,8 @@ def visit_column(element, compiler, **kw):
 
         qry = qry.limit(row_limit)
 
-        if is_timeseries and timeseries_limit and groupby:
+        # Some SQL dialects use timeslot grouping inline with other dimensions
+        if is_timeseries and timeseries_limit and groupby and not time_groupby_inline:
```

Comment:
^ So the previous code rewritten `GROUP BY` as inner query, and then applied ordering and column selection in the outer query. I have zero idea why Vertica requires that for time groupings. It's also slower if the DB is unable to unroll the inner query, so it has to execute the inner query into temporary table and join it with the full table before ordering. CH treats time grouping as any other dimension, so you can use it in `ORDER BY` without problems. The resulting SQL looks just like any other grouping: `SELECT datetime, url, count() as cc FROM table GROUP BY datetime, url ORDER BY datetime, cc`